functions/two_sum.py

```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
```

Comment:
Great with resolving this problem. Look like this On^2 time complexity, whould you try On?